src/axom/primal/tests/primal_curved_polygon.cpp

```diff
@@ -53,8 +53,8 @@ void checkMoments(const primal::CurvedPolygon<CoordType, 2>& bPolygon,
  */
 template <typename CoordType, int DIM>
 primal::CurvedPolygon<CoordType, DIM> createPolygon(
-  const std::vector<primal::Point<CoordType, DIM>> ControlPoints,
-  const std::vector<int> orders)
+  const axom::Array<primal::Point<CoordType, DIM>> ControlPoints,
```

Comment:
Thanks for converting these tests from `std::vector` to `axom::Array` and detailing the missing features described in #869